libs/platform/src/lib/table/components/table-p13-dialog/sorting/sorting.component.ts

```diff
@@ -167,3 +174,11 @@ export class P13SortingDialogComponent implements Resettable {
     /** @hidden */
     private _isRuleValid = (rule: ValidatedSortRule): boolean => rule?.isValid;
 }
+
+@Pipe({ name: 'getAvailableSortColumns', pure: false })
+export class GetAvailableSortColumnsPipe implements PipeTransform {
```

Comment:
is it a specific reason to move pipes inside the components? I'm seeing it in a few places in this PR and it's not the best practise in general

Reply:
https://github.com/SAP/fundamental-ngx/pull/7210#discussion_r763155914